// Edge Case: Nested try/catch statements
int main() {
    try {
        println "Outer try";
        try {
            println "Inner try";
            throw "inner try";
        } catch char* e {
            println "Caught inner:", e;
            throw 1;
        }
    } catch int e {
        println "Caught outer:", e;
    }
    return 0;
}
